scripts/publish.js

```diff
@@ -99,9 +99,6 @@ shell.exec(
   `lerna publish --force-publish --exact --repo-version ${nextVersion} --yes --skip-npm`
 );
 
-// Reformat with prettier since lerna changes the formatting in package.json
-shell.exec('./scripts/reformat-with-prettier.js');
```

Comment:
We tie this now in the top-level package.json to the end of the vscode:package.

@DatGuyJonathan

Reply:
My top level package.json also got formatted during packaging, so I think the reformat script needs to be edited to include top level package.json.

Reply:
Interesting. Sure I can include that.